docs/release-notes/index.md

```diff
@@ -4,6 +4,35 @@
     Please consult [this page](../known-limitations/known-limitations.md) for a list of current limitations
     of the Fabric. Unless otherwise stated, these issues affect all the latest versions of Fabric.
 
+## 25.02
+
+### Highlights
+
+- Celestica DS5000 support as a spine and limited leaf (no local peering supported)
+- MCLAG link state tracking is now enabled (shutdown server-facing MCLAG ports if no spine uplinks are up)
+
+### Tools
+
+- (Experimental) support dump could be created by running `kubectl hhfab support dump`
+  - it includes all regularly requested data from the K8s API, controller logs and etc
```

Comment:
For some reason the rendering on the second line doesn't indent it so it looks like a top level bullet, I think you need more spaces. There is a functional example in the 25.01 notes, tools section.

I would also mention that this feature remove sensitive info.